yetanotherco/aligned_layer
# fix: grafana warnings

# Remove grafana warnings

Merge after #1511 

## Description

Metrics names are not following conventions, so grafana shows a warning about that
ie: received_proofs -> received_proofs_count

## How to Test

1. Run all the components

2. `make run_metrics`

3. Go to grafana dashboard `localhost:3000`

4. Grafana should show metrics correctly and should not show warnings

## Type of change


## Checklist
